Sources/interoperability-tests/InteroperabilityTestsExecutable.swift

```diff
@@ -26,7 +26,7 @@ import NIOPosix
 struct InteroperabilityTestsExecutable: AsyncParsableCommand {
   static var configuration = CommandConfiguration(
     abstract: "gRPC Swift Interoperability Runner",
-    subcommands: [StartServer.self, ListTests.self]
+    subcommands: [StartServer.self, ListTests.self, RunTest.self, RunAllTests.self]
```

Comment:
Rather than having two subcommands, can we just one "RunTests"? If we pass no arguments then it should run all tests, otherwise it should run all specified tests.

This way we have a single subcommand for running tests and more flexibility as we can run whichever tests we like in one go.